.github/workflows/workflows_validator.yml

```diff
@@ -20,12 +20,12 @@ jobs:
       run: pip install boto3
 
     - name: Validate input templates
-      run: python .github/utils/validate_input_templates.py
+      run: python scripts/healthomics_wf/utils/validate_input_templates.py
```

Comment:
after this pr is merged need to update also the validation in terra_pipeline publish:
https://github.com/Ultimagen/terra_pipeline/blob/master/.github/workflows/terra_pipeline-release.yml#L313